hunting-buddy.lic

```diff
@@ -77,6 +77,7 @@ class HuntingBuddy
       stop_on_low_skills = info['stop_on_low']
       stop_on_boxes = info['boxes']
       stop_on_no_moons = info['moons']
+      waiting_room = @settings.safe_room || info['full_waiting_room']
```

Comment:
❓ If I'm reading this right, `safe_room` will take precedence. I would have expected the more specific value, from the hunting info, to be higher priority. Thoughts?

Reply:
Yes, I have it backwards!